Add support for Roborock Zeo (#33583)
home-assistant/home-assistant.github.io: source/_integrations/roborock.markdown

## Dyad entities

> Roborock wet/dry vacuums currently expose some entities through a MQTT connection - it is currently cloud dependent.

Reviewer: It is actually `an`. This abbreviation starts with a vowel sound (`em cue tee tee`), so the correct article is `an` (regardless of the actual first letter in this case).

```suggestion
Roborock wet/dry vacuums currently expose some entities through an MQTT connection - it is currently cloud dependent.
```

Total cleaning time - how long you have cleaned with your wet/dry vacuum.

## Zeo Entities

> Roborock Zeo One currecntly expose some entities through a MQTT connection - it is currently cloud dependent.

Reviewer: ```suggestion
Roborock Zeo One currently exposes some entities through an MQTT connection - it is currently cloud dependent.
```
